### ISSUE TITLE
The 'bootTime' parameter of `vim.vm.RuntimeInfo` at the migrated VMs is unset.

### ISSUE THREAD

**Issue by userlocalhost2000** [vmware/pyvmomi#359]
About the 'bootTime' parameter of `vim.vm.RuntimeInfo` which is got from `vim.VirtualMachine`,
some VMs doesn't have that. And each of them are migrated after the boot.

This problem is occurred on the following environment.
* ESXi: 5.5.0, build: 2068190
* vCenter: 5.5.0, build: 3252642
* pyvmomi: latest of master branch (commit: 4c9debad21bb284c61a64e7c7cb7388b9d0ce4e2)

**Comment by tianhao64** [comment]
@userlocalhost2000 What exactly is the issue? Are those vms powered off? Powered off vm doesn't have the bootTime parameter.

**Comment by userlocalhost2000** [comment]
> Are those vms powered off?

No. These vms are powered on. And this is not a question but a BUG report.
The essence of this issue is that the `bootTime` parameter of each running vms which are migrated will be blank.

I'm not sure either pyvmomi or vCenter causes this problem.

**Comment by tianhao64** [comment]
Is this happening using the UI web client?

**Comment by userlocalhost2000** [comment]
Does `UI web client` means vSphere Web Client?
I can get time that target vm is booted from vSphere Web Client.
